.circleci/config.yml

```diff
@@ -732,6 +732,14 @@ workflows:
           requires:
             - prepare-release
             - production-hold
+      - sync-fundings:
```

Comment:
should these require the release-branch filter like the other jobs above?

Reply:
I don't think so. The `requires` option needs a successful run. If the required task is filtering and won't run, filtering again is a redundancy. At least that's my understanding.

Reply:
I guess the right thing to do would be to drop the redundant `filters`.

Reply:
ah got it, sure makes sense